CHANGES.rst

```diff
@@ -5,6 +5,12 @@
 4.7.1 (unreleased)
```

Comment:
This change should probably increase the minor version as it is a new feature.

Reply:
Done.